tests/scalers/cron/cron_test.go

```diff
@@ -25,8 +25,8 @@ var (
 	scaledObjectName = fmt.Sprintf("%s-so", testName)
 
 	now   = time.Now().Local()
-	start = (now.Minute() + 1)
-	end   = (start + 1)
+	start = (now.Minute() + 1) % 60
```

Comment:
Intentional? Doesn't seem related to the docs

Reply:
It is,  I can open another PR to be more clear. Running e2e test yesterday I found that the generated SO goes from 60 * * * * to 61 * * * * and test failed

Reply:
Let's split it in another PR if OK indeed

Reply:
done! https://github.com/kedacore/keda/pull/3590